Tests/SonarQube.TeamBuild.PreProcessor.Tests/BuildWrapperInstallerTests.cs

```diff
@@ -30,14 +33,17 @@ public void BuildWrapper_PluginNotInstalled_NoDownload()
             BuildWrapperInstaller testSubject = new BuildWrapperInstaller(logger);
 
             // Act
-            testSubject.InstallBuildWrapper(mockServer, rootDir);
+            AnalysisProperties properties = testSubject.InstallBuildWrapper(mockServer, rootDir, "c:\\output\\");
```

Comment:
No new tests, but the existing tests have been updated to check that the properties are set/not set as expected.